README.md

```diff
@@ -5,3 +5,396 @@ Templates rendering plugin support for hapi.js.
 [![Build Status](https://secure.travis-ci.org/hapijs/vision.png)](http://travis-ci.org/hapijs/vision)
 
 Lead Maintainer - [Eran Hammer](https://github.com/hueniverse)
+
+**vision** decorates the [server](https://github.com/hapijs/hapi/blob/master/API.md#server) and
+[reply](https://github.com/hapijs/hapi/blob/master/API.md#reply-interface) interfaces with additional
+methods for managing view engines that can be used to render templated responses. **vision** also
+provides a built-in [handler](https://github.com/hapijs/hapi/blob/master/API.md#serverhandlername-method)
+implementation for creating templated responses.
+
```

Comment:
You should add an example here for registering the plugin since hapi 9 will remove it from the built-in support.